webauthn-rs-core/src/core.rs

```diff
@@ -3056,9 +3108,8 @@ mod tests {
             &RequestRegistrationExtensions::default(),
             false,
         );
-        debug!("{:?}", result);
-        // Currently UNSUPPORTED as openssl doesn't have eddsa management utils that we need.
-        assert!(result.is_err());
+        dbg!("{:?}", &result);
+        assert!(result.is_ok());
```

Comment:
Why does this change to .is_ok() ? We still don't have eddsa?

Reply:
Maybe I missed something but I think the JWT in safetynet attestation now works

Reply:
The JWT library I used supports eddsa

Reply:
Is eddsa required for android?

Reply:
Not sure. I need to dig into this more, but this test flipped after I added safety net support. is there a reason it should fail?

Reply:
Ahhh sorry I mis-understood, it's not a lack of eddsa here, it's ES256, so that's okay.